react-challenge.html

```diff
@@ -23,36 +23,40 @@
     // No need to change anything ABOVE this line ^
 
 
-
+//added brackets to quotation marks for current mood
+//added this.state.moods to this.setState section.
       class App extends React.Component{
         constructor(props){
           super(props)
           this.state = {
             moods: ["😃", "😴", "😍", "🥺", "😡", "🤓", "🤪"],
-            currentMood: ""
+            currentMood: [""]
```

Comment:
Looks like this was changed from string to array of string, which is fine, it just means all the places that need to access currentMood will need to access the array values by using index